sdl2promela/translator.py

```diff
@@ -183,6 +183,18 @@ def __get_state_variable_name(context: Context) -> str:
     )
 
 
+def __get_substate_variable_name(context: Context, substate: str) -> str:
+    return (
+        __GLOBAL_STATE
+        + "."
+        + context.sdl_model.process_name.lower()
+        + "."
+        + substate
+        + SEPARATOR
```

Comment:
Just to confirm - all substates are flattened to a single level, right?

Reply:
yes